Content.Client/GameObjects/Components/Power/DummyWireComponent.cs

```diff
@@ -0,0 +1,48 @@
+#nullable enable
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+using System.Collections.Generic;
+
+namespace Content.Client.GameObjects.Components.Power
+{
+    [RegisterComponent]
+    public class DummyWireComponent : Component
+    {
+        public override string Name => "DummyWire";
+
+        private List<string> _dummyWireProtos = default!;
+
+        private List<IEntity> _dummyWires = new();
+
+        public override void ExposeData(ObjectSerializer serializer)
+        {
+            base.ExposeData(serializer);
+            serializer.DataField(ref _dummyWireProtos, "prototype", new List<string> { "HVDummyWire" });
```

Comment:
if its a list i'd call it prototypes
```suggestion
            serializer.DataField(ref _dummyWireProtos, "prototypes", new List<string> { "HVDummyWire" });
```